lambdas/report-user-identity/src/main/java/uk/gov/di/ipv/core/reportuseridentity/ReportUserIdentityHandler.java

```diff
@@ -141,56 +184,80 @@ private void processUsersToFindLOCAndUpdateDb(List<String> userIds)
                                 String.format("User (%s) VCs not correlated.", userId)));
                 continue;
             }
-            boolean migrated = tacticalVcs.stream().allMatch(vc -> vc.getMigrated() != null);
+            boolean anyVCsMigrated = tacticalVcs.stream().anyMatch(vc -> vc.getMigrated() != null);
+            boolean allVCsMigrated = tacticalVcs.stream().allMatch(vc -> vc.getMigrated() != null);
+
+            if (anyVCsMigrated && !allVCsMigrated) {
+                LOGGER.info(
+                        LogHelper.buildLogMessage(
+                                String.format(
+                                        "Not all VCs are migrated for this user (%s).", userId)));
+            }
             var votAttained =
                     reportUserIdentityService.getStrongestAttainedVotForCredentials(tacticalVcs);
 
             reportUserIdentityItems.add(
                     new ReportUserIdentityItem(
                             userId,
                             votAttained.orElse(Vot.P0).name(),
+                            tacticalVcs.size(),
                             reportUserIdentityService.getIdentityConstituent(tacticalVcs),
-                            migrated));
+                            allVCsMigrated));
         }
         LOGGER.info(LogHelper.buildLogMessage("Updating processed user's identity."));
         reportUserIdentityDataStore.createOrUpdate(reportUserIdentityItems);
     }
 
-    private ReportProcessingResult buildReportProcessingResult() {
+    private ReportProcessingResult.ReportProcessingResultBuilder buildReportProcessingResult(
+            ReportProcessingRequest reportProcessingRequest,
+            ReportProcessingResult.ReportProcessingResultBuilder reportProcessingResult) {
         LOGGER.info(LogHelper.buildLogMessage("Building report processing summary result."));
+        Map<String, AttributeValue> exclusiveStartKey = null;
+        if (reportProcessingRequest.userIdentitylastEvaluatedKey() != null) {
+            exclusiveStartKey = reportProcessingRequest.userIdentitylastEvaluatedKey();
+        }
```

Comment:
Can this just be `var exclusiveStartKey = reportProcessingRequest.userIdentitylastEvaluatedKey()`?

Reply:
done